APE_coordinated_pkg.rst

```diff
@@ -0,0 +1,235 @@
+An Updated Model for the Affiliated Package Ecosystem
+-----------------------------------------------------
+
+author: Erik Tollerud, 2017 Astropy Coordination Meeting Attendees
+
+date-created: 2018 February 22
+
+date-last-revised: 2018 February 22
+
+type: Process
+
+status: Discussion
+
+
+Abstract
+--------
+
+A revision of the concept of "affiliated packages" is proposed.  The idea of
+affiliated packages has continued mostly unaltered since the original Vision
+for Astropy.  However, time has revealed some potential improvements in how the
```

Comment:
Include a link to the vision? (in the astropy docs)

Reply:
It's also linked below, but it doesn't hurt to do it twice!